html/common/coreHead.php

```diff
@@ -476,21 +476,31 @@ function generateNewTag() {
         "class" => "secondary"
     ],
     8 => [
+        "name" => "Closed",
+        "description" => "Pending move to Archive",
+        "foregroundColour" => "#000000",
+        "backgroundColour" => "#F5F5F5",
+        "order" => 8,
+        "assetsAvailable" => false,
+        "isCancelled" => false,
+        "class" => "secondary"
+    ],
+    9 => [
```

Comment:
Won't all the existing values from 7 and grater need to be incremented by one in the database -I.e, we have a project with status id 7 (which previously was closed, will now need to be id 8, as that's closed with these changes. @Jbithell can correct me in his review but I believe that's what the order param in the dict is for - you can tack a new status on the end of the array and adjust the order in them all.

Reply:
Fixed in cb78d71